packages/utilities/gel-foundations/README.md

```diff
@@ -31,18 +31,20 @@ import { css } from 'styled-components';
 
 const SomeStyledComponent = css`
   font-family: ${GEL_FF_REITH_SANS};
+  ${getLongPrimer(script)};
 
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    grid-gap: ${GEL_GUTTER_ABOVE_600PX};
-    ${GEL_BREVIER};
+    grid-gap: ${GEL_GUTTER_ABOVE_600PX};   
   }
 `;
 ```
 
 To allow the typography to be fully accessible and responsive, please note that you should apply a default font-size to the document root (e.g. `html { font-size: 100% }`).
 
-Our typography uses `em` for font-size and `rem` for line-height. `em` allows modularity of components: you can change the component font-size by changing the font-size of its container. `rem` is relative to the document root, so we use that for line-height and spacing for a consistent look-and-feel across the document. You can read our [detailed analysis of "REMs vs EMs for spacing"](https://github.com/bbc/simorgh/blob/latest/docs/Spacing-Units.md) for more information.
+Our typography uses `rem` for font-size and line-height. `rem` is relative to the document root, so we use that for font-size and line-height and spacing for a consistent look-and-feel across the document. You can read our [detailed analysis of "REMs vs EMs for spacing"](https://github.com/bbc/simorgh/blob/latest/docs/Spacing-Units.md) for more information.
 
+You can pass a script name as a prop to the utility typography function (e.g. `getLongPrimer('arabic')`), to get the font-size and line-height for the Typography Style of that script. By default it will use the `news` script as found [here](https://www.bbc.co.uk/gel/guidelines/typography#type-sizes).
```

Comment:
I think this needs updating to the new method of passing script config

Reply:
Thanks for spotting this @dr3. I have updated it.